src/main/java/seedu/address/logic/commands/SortCommand.java

```diff
@@ -0,0 +1,81 @@
+package seedu.address.logic.commands;
+
+import static seedu.address.commons.util.CollectionUtil.requireAllNonNull;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_HEIGHT;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_JERSEY_NUMBER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_PLAYER;
+import static seedu.address.logic.parser.CliSyntax.PREFIX_WEIGHT;
+
+import java.util.Comparator;
+
+import seedu.address.commons.core.Messages;
+import seedu.address.logic.commands.exceptions.CommandException;
+import seedu.address.logic.parser.Prefix;
+import seedu.address.model.Model;
+import seedu.address.model.person.Person;
+
+public class SortCommand extends Command {
+    public static final String COMMAND_WORD = "sort";
+    public static final String MESSAGE_USAGE_PLAYER = COMMAND_WORD
+            + ": To sort player\n"
+            + "Parameters: "
+            + PREFIX_HEIGHT + "SORTING_ORDER or "
+            + PREFIX_WEIGHT + "SORTING_ORDER or "
+            + PREFIX_JERSEY_NUMBER + "SORTING_ORDER\n"
+            + "Example: " + COMMAND_WORD + " " + PREFIX_PLAYER + " " + PREFIX_HEIGHT + "asc";
+    public static final String MESSAGE_USAGE = COMMAND_WORD + ": sort players based on the specified criteria.\n"
+            + MESSAGE_USAGE_PLAYER;
+
+    private final Prefix itemList;
+    private final Prefix sortingCriteria;
+    private final String sortingOrder;
+
+    public SortCommand(Prefix itemList, Prefix sortingCriteria, String sortingOrder) {
```

Comment:
If sort command can only be used to sort player, is prefix P/ still necessary?

Reply:
True that. In that case I will remove the prefix P/